src/report/annotation.go

```diff
@@ -81,3 +85,62 @@ func findingAttributeValue(name string, finding types.ImageScanFinding) string {
 	}
 	return ""
 }
+
+func sortFindings(findings []types.ImageScanFinding) []types.ImageScanFinding {
+	// shallow clone, don't affect source array
+	sorted := slices.Clone(findings)
+
+	// sort by severity rank, then CVE _descending_
+	slices.SortFunc(sorted, func(a, b types.ImageScanFinding) int {
+		sevRank := compareSeverities(string(a.Severity), string(b.Severity))
+		if sevRank != 0 {
+			return sevRank
+		}
+
+		// descending order of CVE, in general this means that newer CVEs will be at
+		// the top
+		return strings.Compare(aws.ToString(b.Name), aws.ToString(a.Name))
```

Comment:
Is this to change from ascending to descending order? If so, `return sevRank * -1` would achieve the same result